cdap-data-fabric/src/main/java/co/cask/cdap/data2/util/hbase/HTableNameConverter.java

```diff
@@ -16,18 +16,87 @@
 
 package co.cask.cdap.data2.util.hbase;
 
+import co.cask.cdap.common.conf.Constants;
+import co.cask.cdap.proto.Id;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Preconditions;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+
 /**
  * Common utility methods for dealing with HBase table name conversions.
  */
 public abstract class HTableNameConverter {
+  protected static final String HBASE_NAMESPACE_PREFIX = "cdap_";
```

Comment:
There's a JIRA for that, separate from the subject of this PR: https://issues.cask.co/browse/CDAP-1670

So it will get done in a separate PR by either Bhooshan or me.